packages/account-abstraction/src/TransactionDetailsForUserOp.ts

```diff
@@ -19,4 +19,7 @@ export interface TransactionDetailsForBatchUserOp {
   gasLimit?: BigNumberish
   maxFeePerGas?: BigNumberish
   maxPriorityFeePerGas?: BigNumberish
+  paymasterServiceData?: {
+    webhookData?: object
```

Comment:
you can define type then PaymasterServiceData
and webhookData can be specific type than just object if it can be made generic